src/test/app/PayStrand_test.cpp

```diff
@@ -857,6 +844,30 @@ struct PayStrand_test : public beast::unit_test::suite
             }
         }
     }
+    void
```

Comment:
FYI, there are five `bool`s constructed in `for` loops that could be declared `const` on lines 750 through 776.  No change necessary...